tabbycat/tournaments/views.py

```diff
@@ -84,12 +85,11 @@ def get_context_data(self, **kwargs):
                     'content_object', 'user').order_by('-timestamp')[:updates]
         kwargs["initialActions"] = json.dumps([a.serialize for a in actions])
 
-        subs = BallotSubmission.objects.filter(
-            debate__round__tournament=t, confirmed=True).prefetch_related(
-            'teamscore_set__debate_team',
-            'teamscore_set__debate_team__team').select_related(
-            'debate__round__tournament').order_by('-timestamp')[:updates]
-        subs = [bs.serialize_like_actionlog for bs in subs]
+        subs = t.current_round.debate_set.filter(
```

Comment:
I'd give this variable a different name (like `debates`) to be clearer about what it is
(then still call the new variable `subs` in line 92)